admin/class-perfecty-push-admin.php

```diff
@@ -95,6 +97,8 @@ public function enqueue_scripts() {
 
 		wp_enqueue_script( $this->plugin_name, plugin_dir_url( __FILE__ ) . 'js/perfecty-push-admin.js', array( 'jquery', 'wp-i18n' ), $this->version, false );
 		wp_enqueue_script( 'chartjs', plugin_dir_url( __FILE__ ) . 'js/chart.bundle.min.js', array( 'jquery' ), $this->version, false );
+		wp_enqueue_script( 'jquery-timepicker', plugin_dir_url( __FILE__ ) . 'js/jquery.timepicker.min.js', array( 'jquery' ), '1.3.5', false );
```

Comment:
I'll fix this, but for future versions we should use `$this->version`, we use the Plugin version to expire the cached assets (JS/CSS), so in the next version it will append `1.3.0` to the url and no hard refresh will be required to get the latest JS changes after the upgrade.